src/main/java/seedu/address/storage/JsonAdaptedClient.java

```diff
@@ -147,24 +147,28 @@ public Client toModelType() throws IllegalValueException {
             throw new IllegalValueException(
                     String.format(MISSING_FIELD_MESSAGE_FORMAT, ContractExpiryDate.class.getSimpleName()));
         }
-        // It's possible for contractExpiryDate to have an empty String as a value, which would indicate
-        // the client has no contractExpiryDate set, and {@code ParserUtil.parseContractExpiryDate} will
-        // parse it into
-        // a {@code ContractExpiryDate.NULL_DATE}
+        /*
+        * It's possible for {@code contractExpiryDate} to have an empty String as a value, which would indicate
+        * the client has no contractExpiryDate set, and {@code ParserUtil.parseContractExpiryDate} will
+        * parse it into a {@code ContractExpiryDate.NULL_DATE}
+        * */
         if (!contractExpiryDate.isEmpty() && !ContractExpiryDate.isValidDate(contractExpiryDate)) {
             throw new IllegalValueException(ContractExpiryDate.MESSAGE_CONSTRAINTS);
         }
         final ContractExpiryDate modelContractExpiryContractExpiryDate =
                 ParserUtil.parseContractExpiryDate(contractExpiryDate);
-        // Does not throw an exception if lastModifiedInstant is missing/invalid due to corruption of data.
-        // This field is merely metadata for us and is not significant enough to discard client's data due
-        // to this field being missing.
+        /*
+        * Does not throw an exception if lastModifiedInstant is missing/invalid due to corruption of data.
+        * This field is merely metadata for us and is not significant enough to discard client's data due
+        * to this field being missing.
+        * */
```

Comment:
Block comments should follow this format:
![image](https://user-images.githubusercontent.com/46164638/98256111-24b22e80-1fb9-11eb-864a-bad6eea80428.png)

Reply:
changed it!

![image](https://user-images.githubusercontent.com/38996397/98257172-54ae0180-1fba-11eb-979a-e9d36054984d.png)